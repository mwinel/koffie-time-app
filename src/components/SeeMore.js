import React from "react";
import { ButtonPrimary } from "../styles/ButtonPrimary";

const SeeMore = () => {
  return <ButtonPrimary outline>See More</ButtonPrimary>;
};

export default SeeMore;
