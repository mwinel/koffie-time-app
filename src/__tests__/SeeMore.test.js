import React from "react";
import { render, unmountComponentAtNode } from "react-dom";
import { act } from "react-dom/test-utils";
import pretty from "pretty";
import SeeMore from "../components/SeeMore";

let container = null;

beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  // cleanup on exiting
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

it("renders footer component", () => {
  act(() => {
    render(<SeeMore />, container);
  });
  expect(container.innerHTML).toBeDefined();
  expect(container.innerHTML).toContain("See More");
  expect(pretty(container.innerHTML)).toMatchSnapshot();
});
