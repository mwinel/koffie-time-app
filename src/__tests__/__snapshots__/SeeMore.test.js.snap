// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer component 1`] = `"<button class=\\"sc-bdVaJa gvYZDs\\">See More</button>"`;
